corehq/apps/performance_sms/dbaccessors.py

```diff
@@ -23,9 +21,9 @@ def by_interval(interval_keys):
 def delete_all_configs():
     assert settings.UNIT_TESTING
     db = PerformanceConfiguration.get_db()
-    all_docs = [row['doc'] for row in db.view(
-        'performance_sms/by_domain',
-        reduce=False,
-        include_docs=True
-    )]
-    db.bulk_delete(all_docs)
+
+    # Since this is a unit test, number of domains is small
```

Comment:
Looks like this function could be called outside of a unit test. Might want to add the "unit test only" decorator (I don't recall the exact decorator name).